main.tf

```diff
@@ -667,8 +668,9 @@ resource "aws_pipes_pipe" "this" {
   source = each.value.source
   target = each.value.target
 
-  description   = lookup(each.value, "description", null)
-  desired_state = lookup(each.value, "desired_state", null)
+  kms_key_identifier = var.kms_key_identifier
```

Comment:
Shouldn't it be using lookup and fetch from each like the rest of arguments? I think it should in all places.

Reply:
Done, please review.